target/linux/ramips/image/mt7620.mk

```diff
@@ -495,7 +495,7 @@ define Device/hootoo_ht-tm05
   DEVICE_MODEL := HT-TM05
   DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-i2c-ralink
   LOADER_TYPE := bin
-  LOADER_FLASH_OFFS := 0xFD200000
+  LOADER_FLASH_OFFS := 0x200000
```

Comment:
Hi!
I don't like adding improper value in device support commit and change that in a following one.
I'd prefer reorder the commits so that the imporper FLASH_OFFS doesn't appear in the final commit log.

let me get this loader stuff fixed for ramips (including mt7621) myself today and you could base your device support on that.

Reply:
> let me get this loader stuff fixed for ramips (including mt7621) myself today

FYI: that weird bootloader on DIR-860L ... :
```
3: System Boot system code via Flash.
## Booting image at bfc50000 ...
addr:80500000
We have SEAMA, Image Size = 5111748
Verifying Checksum ...
Uncompressing SEAMA linux.lzma ... OK
## Transferring control to Linux (at address 00000000) ...
## Giving linux memsize in MB, 128

Starting kernel ...



OpenWrt kernel loader for MIPS based SoC
Copyright (C) 2011 Gabor Juhos <juhosg@openwrt.org>
Looking for OpenWrt image... not found at 0xb0000000
not found at 0xb0001000
not found at 0xb0002000
not found at 0xb0003000
.
.
.

```
... finds twice the OKLI magic:
* `0xb0501000`
* `0xb8501000`

Skipping those addresses it hangs at `0xbe10d000` + `0x1000` = `0xbe10e000`.

Reply:
> let me get this loader stuff fixed for ramips (including mt7621) myself today and you could base your device support on that.

Sure, NP! But I think this is about git, and not having the commit showing the incorrect value (rather, only show the final answer, right?). My apologies, but I may need a bit of git help here, not sure how to make that original "go away". Any pointers?

Thanks!!!

Reply:
@arrmo: just drop your loader stuff commit, when 981213 is ready!

Reply:
Ahh, OK - I mis-read. So @981213 will add this in (more generically), and then I can remove that lzma-loader (second?) commit ... right? May need some help stepping through that process, but hopefully then we can get this one all approved also 😄.

Thanks!